repo/blob/filesystem/filesystem_storage_capacity_openbsd.go

```diff
@@ -0,0 +1,30 @@
+//go:build openbsd
+// +build openbsd
+
+package filesystem
+
+import (
+	"context"
+	"syscall"
+
+	"github.com/pkg/errors"
+
+	"github.com/kopia/kopia/internal/retry"
+	"github.com/kopia/kopia/repo/blob"
+)
+
+func (fs *fsStorage) GetCapacity(ctx context.Context) (blob.Capacity, error) {
+	c, err := retry.WithExponentialBackoff(ctx, "GetCapacity", func() (interface{}, error) {
+		var stat syscall.Statfs_t
+		if err := syscall.Statfs(fs.RootPath, &stat); err != nil {
+			return blob.Capacity{}, errors.Wrap(err, "GetCapacity")
+		}
+
+		return blob.Capacity{
+			SizeB: uint64(stat.F_blocks) * uint64(stat.F_bsize), // nolint:unconvert,nolintlint
```

Comment:
i'm surprised that `nolint` works, actually, but if the tests passes, we can take it.

Reply:
Yes, we can, with a couple of caveats:
1. If any syscalls fail the program will crash. I took the more verbose approach to be able to handle such errors gracefully. What do you think is the more appropriate way to handle errors here?
2. In the linked document, `syscall.StringToUTF16Ptr()` is deprecated. I'd use `syscall.UTF16PtrFromString()` instead.